devtools/cli/src/main/java/io/quarkus/cli/ProjectExtensions.java

```diff
@@ -11,7 +11,8 @@
 
 @CommandLine.Command(name = "extension", aliases = {
         "ext" }, sortOptions = false, mixinStandardHelpOptions = false, description = "List, add, and remove extensions of an existing project.", subcommands = {
-                ProjectExtensionsAdd.class, ProjectExtensionsList.class, ProjectExtensionsRemove.class })
+                ProjectExtensionsAdd.class, ProjectExtensionsList.class, ProjectExtensionsCategories.class,
```

Comment:
what should be the order here? If we stick with alphabetical, it should be [Add, Categories, List, Remove] ..  If we wanted to group list/categories first: [List, Categories, Add, Remove] might be better?

Reply:
Yeah, I think both options are better then the current order. I will go with [List, Categories, Add, Remove].

Help text looks like:

```
Usage: quarkus extension [-eh] [--[no-]registry-client] [--verbose] [COMMAND]
List, add, and remove extensions of an existing project.
...
Commands:
  list, ls         List platforms and extensions for this project.
  categories, cat  List existing extension categories.
  add              Extension(s) to add to this project.
  remove, rm       Remove extension(s) from this project.
```